### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by ganheather02**
added get and update methods

Closes #[ISSUE-NUM].

### CODE REVIEW

**Review by reyna-abhyankar — CHANGES_REQUESTED** [review]
Code isn't compiling - says items is not defined

File: app/routes/menu/index.js [hunk 1]
```diff
@@ -34,17 +34,25 @@ router.get('/', (req, res, next) => {
...
-// Post menu request, add to cart.
-router.post('/', (req, res) => {
+function getCart(req) {
   try {
-    // Attempt to push new order into cart.
     const { cart } = req.cookies;
-    cart.push(req.body);
-    res.cookie('cart', cart);
+    return cart;
   } catch (TypeError) {
```
Comment (declansullivan) [inline app/routes/menu/index.js:41]: Need to change this to an if/else.

**Review by declansullivan — CHANGES_REQUESTED** [review]
Adding the first item to the cart fails, this is because the TypeError we were getting in the past was when we were trying to push to undefined.  const { cart } = req.cookies does not cause an error, it only returns undefined, which causes problems down the road.  This should be a quick fix we can make and finalize by the end of the meeting

**Comment by declansullivan** [comment]
Merged this PR into my post request PR

**Pull request closed without merging**